apps/console/src/features/users/components/wizard/bulk-import-user-wizard.tsx

```diff
@@ -726,6 +792,90 @@ export const BulkImportUserWizard: FunctionComponent<BulkImportUserInterface> =
         };
     };
 
+    /**
+     * Generate SCIM Bulk Request Body
+     *
+     * @param attributeMapping - attribute mapping.
+     * @returns SCIMBulkRequestBody
+     */
+    const generateMultipleUsersSCIMRequestBody = (): SCIMBulkEndpointInterface => {
+        // Create the data operations
+        const operations: SCIMBulkOperation[] = [];
+
+        // Create the user record
+        emailData.map((email: string) => {
+            const userDetails: UserDetailsInterface = {
+                emails: [
+                    {
+                        primary: true,
+                        value: email
+                    }
+                ],
+                schemas: [
+                    "urn:ietf:params:scim:schemas:core:2.0:User",
+                    "urn:ietf:params:scim:schemas:extension:enterprise:2.0:User"
+                ],
+                userName: 
+                    userstore.toLowerCase() !== PRIMARY_USERSTORE.toLowerCase()
+                        ? `${userstore}/${email}`
+                        : email,    
+                [ userstore.toLowerCase() !== PRIMARY_USERSTORE.toLowerCase()
+                    ? UserManagementConstants.CUSTOMSCHEMA
+                    : UserManagementConstants.ENTERPRISESCHEMA
+                ]: {
+                    askPassword: "true"
+                }
+            };
+            const asyncOperationID: string = uuidv4();
```

Comment:
Do we really need to send random IDs in API payloads? AFAIK, this should be something handled from the backend.

Reply:
This ID will be used to make a reference to another when we are introducing the roles in the future.